packages/core/src/App/Containers/AccountSignupModal/account-signup-modal.jsx

```diff
@@ -34,12 +34,18 @@ const AccountSignup = ({
     const history_value = React.useRef();
     const [pw_input, setPWInput] = React.useState('');
     const [is_password_modal, setIsPasswordModal] = React.useState(false);
+    const isCountryScreenLoggedOnceRef = React.useRef(false);
     const [is_disclaimer_accepted, setIsDisclaimerAccepted] = React.useState(false);
     const [is_questionnaire, setIsQuestionnaire] = React.useState(false);
     const [ab_questionnaire, setABQuestionnaire] = React.useState();
     const [modded_state, setModdedState] = React.useState({});
     const language = getLanguage();
 
+    const [is_signup_flow_error] = useGrowthbookGetFeatureValue({
```

Comment:
What's the expected behaviour of this feature flag?
I dont understand it.

Reply:
Naming is also vague.

Reply:
I think it's like `enable_signup_error_tracking`

Reply:
Its like should we enable the `signup_flow_error` feature flag or not